src/debug/debug-engine.js

```diff
@@ -119,6 +119,16 @@ export class DebugEngine {
         return response.body;
     }
 
+    async evaluate(expression, stackFrameId) {
+        const response = await this.debugClient.evaluateRequest({
+            expression: expression,
+            frameId: stackFrameId,
+            context: "watch"
```

Comment:
context can be changed, so keep the context in method parameters, don't hard-code it here.